examples/pipelines/ccfraud/submit.py

```diff
@@ -122,6 +122,13 @@ def connect_to_aml():
     return ML_CLIENT
 
 
+#############################################
+### GET ML_CLIENT AND COMPUTE INFORMATION ###
+#############################################
+
+ML_CLIENT = connect_to_aml()
```

Comment:
IIRC @garg-amit  moved connect_to_aml() below for some reason. I understand this will be problematic for your use case, let's clarify why that was moved.

Reply:
That was moved in order to resolve the issue we faced when preserving the precedence hierarchy (PR #209 ). If a user wishes to validate the pipeline without really submitting it, then we don't need to create an ML Client obj.

Reply:
Thanks @garg-amit that's reasonable. However, I believe this cannot be applied when searching for specs for compute ahead of time.

Reply:
Note: We allow users to validate a pipeline without providing rg name, subscription id, etc up front. So, if you wish to create an ML client obj here, it would throw an error if those details are not provided since MLClient obj cannot be created with "None" values.